mne_qt_browser/_pg_figure.py

```diff
@@ -36,11 +36,11 @@
 from pyqtgraph import (AxisItem, GraphicsView, InfLineLabel, InfiniteLine,
                        LinearRegionItem, PlotCurveItem, PlotItem,
                        Point, TextItem, ViewBox, mkBrush,
-                       mkPen, setConfigOption, mkQApp, mkColor)
+                       mkPen, setConfigOption, mkColor)
 from scipy.stats import zscore
 
 from mne.viz import plot_sensors
-from mne.viz.backends._utils import _init_qt_resources
+from mne.viz.backends._utils import _init_mne_qtapp
```

Comment:
This is going to be a problem for people on 0.24. We won't backport the `_init_mne_qtapp` since it's an enhancement, so you'll have to `try/except` it here for now.

One way to take care of this is to try to import, and if it fails, just have a copy-paste copy here for a couple of months until we release 1.0 (and 0.24 is thus no longer supported).

Reply:
Ah ok, I didn't know that backport-policy.

Reply:
Yeah we have to make sure that a new release of mne-qt-browser works for both people on stable MNE (currently 0.24) and latest dev MNE (1.0.dev0). Hopefully the CIs already have this setup, in theory they should catch this error!